django/template/defaulttags.py

```diff
@@ -49,6 +49,8 @@ def render(self, context):
 
 
 class CsrfTokenNode(Node):
+    child_nodelists = ()
```

Comment:
Can we add the same to the `CommentNode`?

Reply:
Good call, that's probably common enough (in my templates anyway!) that it's worth having.